tests/MyGame/Example/Monster.go

```diff
@@ -231,6 +265,14 @@ func (rcv *Monster) Testbool() byte {
 	return 0
 }
 
+func (rcv *Monster) MutateTestbool(n byte) bool {
+	o := flatbuffers.UOffsetT(rcv._tab.Offset(34))
+	if o != 0 {
+		 return rcv._tab.MutateByte(o + rcv._tab.Pos, n)
```

Comment:
I think we can use [these methods](https://github.com/google/flatbuffers/pull/259/files#diff-52e6c978a6a92f3ef383e2da6f7b26d1R447).